_includes/youtube-widget.html

```diff
@@ -0,0 +1,3 @@
+<h3>Youtube</h3>
```

Comment:
I think we should update this title to **YouTube**. I know it doesn't have impact today, since we're uppercasing titles, but for future changes, I think we should keep it right.

```suggestion
<h3>YouTube</h3>
```